packages/big-design/src/components/Dropdown/spec.tsx

```diff
@@ -329,3 +329,19 @@ test("doesn't render tooltip on enabled item", () => {
 
   expect(queryByText(tooltipText)).not.toBeInTheDocument();
 });
+
+test('no errors expected if all options are disabled', () => {
+  const { getByRole } = render(
+    <Dropdown
+      onClick={onClick}
+      options={[
+        { content: 'Option 1', value: '0', disabled: true },
+        { content: 'Option 2', value: '1', disabled: true },
+      ]}
+      trigger={<Button>Button</Button>}
+    />,
+  );
+
+  const trigger = getByRole('button');
+  fireEvent.click(trigger);
```

Comment:
Can you use `toThrow` to better capture what we are trying to test? https://jestjs.io/docs/en/expect#tothrowerror

Reply:
I looked for something like this and somehow missed it. 👍